### ISSUE TITLE
GitHub RSA SSH Key Problem? 

### ISSUE THREAD

**Issue by simon-wolf** [elixir-cldr/cldr#195]
Hi Kip and thank you for your great work on this project, it is very much appreciated.

I'm having a problem today where compiling my project effectively stops when it is trying to generate locales and I am wondering if it is due to GitHub updating their RSA SSH host key:

https://github.blog/2023-03-23-we-updated-our-rsa-ssh-host-key/ 

the output I get from `mix.compile` is:

```
$ mix compile
Compiling 153 files (.ex)
Generating Mindme.Cldr for 2 locales named [:en, :und] with a default locale named :en
Compiling lib/mindme/cldr.ex (it's taking more than 10s)
```

And then it just sits there and the only think I can think of is that the key change has caused a problem. Maybe. :man_shrugging:

**Comment by kipcole9** [comment]
@simon-wolf thanks for the kind words. And very sorry for the inconvenience. I just tried on my dev machine and locales downloaded fine. The blog post does say it should only affect ssh gitops which I understand to mean it shouldn't affect https / tls.

TLDR; I don't know what the issue might be. I don't set either `:timeout` or `:connection_timeout` on the `:https` requests so they default to infinity which isn't a great idea (and I'll change  that for the upcoming release so its good timing on that count).

One thing you can try is:
```bash
wget https://raw.githubusercontent.com/elixir-cldr/cldr/v2.36.0/priv/cldr/locales/en.json
```
Which is basically what is happening under the covers. Let me know if that works?

**Comment by simon-wolf** [comment]
@kipcole9 Yes, that works.

Apologies then because I don't think this is a problem with the package itself. It is entirely possible I've broken something somewhere else in my project. And having re-read things, the `en` locate should be available anyway via the Hex package which would mean it is not a network issue. I'll keep digging.

I'll close the issue and thank you for your response and adding a timeout in the upcoming release which would be great.

**Comment by simon-wolf** [comment]
Thanks Kip. running `mix deps.compile ex_cldr --force && mix compile --force` seemed to unblock things although before that I had deleted the `_build` folder so it's all a bit strange.

Sorry for interrupting your evening/night and thank you again.

**Comment by kipcole9** [comment]
Glad you're back up and running.  Don't hesitate to reopen if it crops up again.
